### PR TITLE
Addressed comments from Jorge in PR #3419

### PR DESCRIPTION
**Pull request by tapadipti**
Files changed (all inside `content/docs/studio/`) as per comments in #3419 :

- get-started.md
- user-guide/account-management.md
- user-guide/connect-custom-gitlab-server.md
- user-guide/teams.md
- user-guide/views/create-view.md

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Checking now. Sorry for the delay!

File: content/docs/studio/get-started.md [hunk 1]
```diff
@@ -9,13 +9,9 @@ https://www.youtube.com/watch?v=hKf4twg832g
    using your GitHub.com, GitLab.com or Bitbucket.org account, or your email
    address.
 
-   <admon type="info">
-
    When you sign up with your email address, Studio will send you a verification
    email before you can login.
```
Comment (jorgeorpinel) [inline content/docs/studio/get-started.md:14]: My comment was that this is standard/ expected. Probably no need to even mention it?

```suggestion
```

File: content/docs/studio/user-guide/views/create-view.md [hunk 2]
```diff
@@ -6,13 +6,9 @@ To create a new view, follow these steps.
    using your GitHub.com, GitLab.com or Bitbucket.org account, or your email
    address.
 
-   <admon type="info">
-
    When you sign up with your email address, Studio will send you a verification
    email before you can login.
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/views/create-view.md:11]: Same

```suggestion
```

File: content/docs/studio/user-guide/account-management.md [hunk 3]
```diff
@@ -24,7 +24,10 @@ You can update your username and password.
 **Managing email addresses:**
 
 You can add multiple email addresses to a single Studio account. You can login
-to the account with any of your verified email addresses.
+to the account with any of your verified email addresses. This is true even if
+you signed up using your GitHub, GitLab or Bitbucket accounts (instead of email
+sign-up). You can use any of your registered email addresses to sign in to
+Studio as long as you have set up a password for your account.
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/account-management.md:30]:
```suggestion
to the account with any of your verified email addresses
as long as you have set up a password for your account. This is true even if
you signed up using your GitHub, GitLab, or Bitbucket.
```

File: content/docs/studio/user-guide/account-management.md [hunk 4]
```diff
@@ -40,21 +43,19 @@ In this section, you can,
...
   this section. If you want to connect to a self-hosted GitLab server, you
-  should create a team and set up the GitLab server connection in the team
-  settings. For more details, refer
-  [here](/doc/studio/user-guide/connect-custom-gitlab-server).
+  should create a team and
+  [set up the GitLab server connection](/doc/studio/user-guide/connect-custom-gitlab-server)
+  in the team settings.
 
-- Disconnect from your GitHub.com, GitLab.com or Bitbucket.org accounts.
+- Disconnect from your GitHub, GitLab or Bitbucket accounts.
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/account-management.md:58]:
```suggestion
- Disconnect from your GitHub, GitLab, or Bitbucket accounts.
```

File: content/docs/studio/user-guide/connect-custom-gitlab-server.md [hunk 5]
```diff
@@ -34,11 +34,11 @@ you have any trouble setting up the connection to your GitLab server, please
 <admon type ="info">
 
 Connecting to a self-hosted GitLab server is different from connecting to your
-account on GitLab SaaS ([GitLab.com](http://gitlab.com/)). To connect to
-[gitlab.com](http://gitlab.com/) go to the `Git integrations` section in your
+account on GitLab SaaS ([gitlab.com](http://gitlab.com/)). To connect to
+`gitlab.com` go to the `Git integrations` section in your
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/connect-custom-gitlab-server.md:38]:
```suggestion
GitLab, go to the `Git integrations` section in your
```

File: content/docs/studio/user-guide/connect-custom-gitlab-server.md [hunk 6]
```diff
@@ -34,11 +34,11 @@ you have any trouble setting up the connection to your GitLab server, please
...
 [personal profile settings](https://studio.iterative.ai/user/_/profile). If you
 signed up to use Iterative Studio using your GitLab account, you may already
-have a connection to [gitlab.com](http://gitlab.com/).
+have a connection to `gitlab.com`.
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/connect-custom-gitlab-server.md:41]:
```suggestion
have the connection.
```

**Review by jorgeorpinel — APPROVED** [review]
Left some more suggestions. Up to you @tapadipti ☝🏼

File: content/docs/studio/user-guide/connect-custom-gitlab-server.md (hunk 5 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/connect-custom-gitlab-server.md]: Changed it to "GitLab SaaS"

**Pull request merged**